tests/tests_data/streaming/test_data_optimizer.py

```diff
@@ -207,7 +279,7 @@ def prepare_item(self, item):
 
 @pytest.mark.parametrize("delete_cached_files", [False, True])
 @pytest.mark.parametrize("fast_dev_run", [False, True])
-@pytest.mark.skipif(condition=True, reason="Requires: ['pil']")
+@pytest.mark.skipif(condition=not _PIL_AVAILABLE or sys.platform == "win32", reason="Requires: ['pil']")
```

Comment:
did you check if the test runs in our CI (pillow being available?)

Reply:
<img width="826" alt="Screenshot 2023-10-26 at 17 05 08" src="https://github.com/Lightning-AI/lightning/assets/12861981/ad88b305-d5bf-4cc5-b989-a9624d90263f">

Reply:
Yes, it is there.